dotcom-rendering/src/web/layouts/NewsletterSignupLayout.tsx

```diff
@@ -0,0 +1,232 @@
+import {
+	brandBackground,
+	brandBorder,
+	brandLine,
+} from '@guardian/source-foundations';
+import type { ArticleFormat } from '@guardian/libs';
+
+import { StraightLines } from '@guardian/source-react-components-development-kitchen';
+import { css } from '@emotion/react';
+import { buildAdTargeting } from '../../lib/ad-targeting';
+import { Header } from '../components/Header';
+import { Footer } from '../components/Footer';
+import { SubNav } from '../components/SubNav.importable';
+import { ElementContainer } from '../components/ElementContainer';
+import { HeaderAdSlot } from '../components/HeaderAdSlot';
+import { MobileStickyContainer, AdSlot } from '../components/AdSlot';
+import { Nav } from '../components/Nav/Nav';
+import { getCurrentPillar } from '../lib/layoutHelpers';
+import { Stuck, BannerWrapper } from './lib/stickiness';
+import { Island } from '../components/Island';
+import { OnwardsUpper } from '../components/OnwardsUpper.importable';
+import { NewsLetterSignupBanner } from '../components/NewsLetterSignupBanner';
+import { getContributionsServiceUrl } from '../lib/contributions';
+import { ArticleContainer } from '../components/ArticleContainer';
+import { ArticleBody } from '../components/ArticleBody';
+import { ContainerLayout } from '../components/ContainerLayout';
+
+interface Props {
+	CAPIArticle: CAPIArticleType;
+	NAV: NavType;
+	format: ArticleFormat;
+	palette: Palette;
```

Comment:
Tiny thing: There was a [recent refactor](https://github.com/guardian/dotcom-rendering/pull/4379) to have `palette` created locally, rather than passed in as a prop, can we do that here too, please?

Reply:
sure, no problem